modulus/datapipes/healpix/data_modules.py

```diff
@@ -212,7 +212,7 @@ def open_time_series_dataset_classic_prebuilt(
     if not ds_path.exists():
         raise FileNotFoundError(f"Dataset doesn't appear to exist at {ds_path}")
 
-    result = xr.open_zarr(ds_path, chunks={"time": batch_size})
+    result = xr.open_zarr(ds_path)
```

Comment:
Removing this chunks argument is reasonable, but what were the specific reasons for that here?

Reply:
Not that I could find, I suspect this was hardcoded for a specific dataset.